module/documents/actor.mjs

```diff
@@ -374,71 +362,65 @@ export default class ActorEd extends Actor {
-      const knockdownAbilityId = await KnockDownItemsPrompt.waitPrompt( knockdownAbilities );
+    const knockdownAbility = await fromUuid(
+      await this.getPrompt( "knockDown" )
+    );
 
-      if ( knockdownAbilityId ) {
-        const selectedAbility = knockdownAbilities.find( ability => ability.id === knockdownAbilityId );
-        if ( selectedAbility ) {
-          const { attribute, level, devotionRequired: devotion, strain: abilityStrain } = selectedAbility.system;
-          knockdownStep = attributes[attribute].step + level;
-          devotionRequired = devotion ? true : false;
-          strain = { base: abilityStrain };
-        }
-      }
+    if ( knockdownAbility ) {
+      const { attribute, level, devotionRequired: devotion, strain: abilityStrain } = knockdownAbility.system;
+      knockdownStep = attributes[attribute]?.step || knockdownStep + level;
+      devotionRequired = !!devotion;
+      strain = { base: abilityStrain };
     }
-    const difficulty = amount > 0 ? amount - characteristics.health.woundThreshold : 0;
-    const difficultyFinal = { base: difficulty };
+
+    const difficultyFinal = {
+      base: Math.max( damageTaken - characteristics.health.woundThreshold, 0 ),
+    };
     const chatFlavor = game.i18n.format( "ED.Chat.Flavor.knockdownTest", {
       sourceActor: this.name,
       step: knockdownStep
     } );
 
-    const knockdownStepFinal = { base: knockdownStep + this.system.singleBonuses.knockdownEffects.value };
+    const knockdownStepFinal = {
+      base: knockdownStep,
+      modifiers: {
+        "localize: Global Knockdown Bonus": this.system.singleBonuses.knockdownEffects.value,
+      }
+    };
     const edRollOptions = EdRollOptions.fromActor(
       {
-      testType: "action",
-      rollType: "knockdown",
-      strain: strain,
-      target: difficultyFinal,
-      step: knockdownStepFinal,
-      devotionRequired: devotionRequired,
-      chatFlavor: chatFlavor,
+        testType: "action",
+        rollType: "knockdown",
+        strain: strain,
+        target: difficultyFinal,
+        step: knockdownStepFinal,
+        devotionRequired: devotionRequired,
+        chatFlavor: chatFlavor
       },
-      this,
-     );
+      this
+    );
     const roll = await RollPrompt.waitPrompt( edRollOptions, options );
 
     this.#processRoll( roll );
   }
 
   async jumpUp( options = {} ) {
-    if ( this.system.condition.knockedDown === false ) {
-      ui.notifications.warn( "Localize: You are not knocked down." );
+    if ( !this.system.condition.knockedDown ) {
+      ui.notifications.warn( "LocalizeLabel: You are not knocked down.", { localize: true } );
       return;
     }
     const { attributes } = this.system;
     let devotionRequired = false;
     let strain = 2;
     let jumpUpStep = attributes.dex.step;
 
-    const jumpUpAbilities = this.items.filter( item => item.system.edid === "jump-up" );
-
-    if ( jumpUpAbilities.length > 0 ) {
```

Comment:
I don't get it. If your ability has an attribute assigned it should not take the jump up step level, that's correct. I assume your abilities all have an attribute value?
This is just a failsafe in case there is an ability that doesn't rely on an attribute and just add its level flat on the test...